tests/cephfs/nfs/nfs_info_cluster_id_and_ls_export_detailed.py

```diff
@@ -0,0 +1,131 @@
+import json
+import random
+import string
+import traceback
+
+from ceph.ceph_admin import CephAdmin
+from tests.cephfs.cephfs_utilsV1 import FsUtils
+from utility.log import Log
+
+log = Log(__name__)
+"""
+pre-requisites:
+1. nfs cluster ready
+2. export the nfs cluster
+Test operation:
+1. check if input and --detailed value match
+2. check if nfs clsuter info using its cluster_id
+"""
+
+
+def run(ceph_cluster, **kw):
+    try:
+        tc = "CEPH-83573992"
+        log.info(f"Running CephFS tests for BZ-{tc}")
+        fs_util = FsUtils(ceph_cluster)
+        config = kw.get("config")
+        build = config.get("build", config.get("rhbuild"))
+        clients = ceph_cluster.get_ceph_objects("client")
+        client1 = clients[0]
+        fs_details = fs_util.get_fs_info(client1)
+        if not fs_details:
+            fs_util.create_fs(client1, "cephfs")
+        fs_util.auth_list([client1])
+        fs_util.prepare_clients(clients, build)
+
+        instance = CephAdmin(cluster=ceph_cluster, **config)
+        host_cmd = "cephadm shell ceph orch ps --format json-pretty"
+        res, ec = instance.installer.exec_command(sudo=True, cmd=host_cmd)
+        results = json.loads(res)
+        nfs_hosts_name = []
+        log.info(print(results))
+        nfs_target_node = ""
+        for ps in results:
+            log.info(print(ps))
+            if "nfs" in ps["service_name"]:
+                nfs_hosts_name.append(ps["hostname"])
+            if "mds" in ps["service_name"]:
+                nfs_target_node = ps["hostname"]
+        log.info(print("NFS service is being hosted by ", nfs_hosts_name))
+
+        rand = "".join(
+            random.choice(string.ascii_lowercase + string.digits)
+            for _ in list(range(5))
+        )
+        cluster_id = f"test-nfs_{rand}"
+        path = "/"
+        export_id = "1"
+        fs_name = "cephfs"
+        bind = "/ceph"
+        user_id = f"nfs.{cluster_id}.{export_id}"
+
+        passed = []
+        client1.exec_command(
+            sudo=True, cmd=f'ceph nfs cluster create {cluster_id} "{nfs_target_node}"'
+        )
+        cmd_nfs_export = (
+            f"ceph nfs export create cephfs {cluster_id} {bind} {fs_name} --path={path}"
+        )
+
+        client1.exec_command(sudo=True, cmd=cmd_nfs_export)
+
+        res, ec = client1.exec_command(
+            sudo=True, cmd=f"ceph nfs export ls {cluster_id} --detailed --format json"
+        )
+        results = json.loads(res)
```

Comment:
can we log results for quick reference in future

Reply:
@AmarnatReddy I have logged the results

Reply:
@julpark-rh - Can you add the logs link